src/client/Microsoft.Identity.Client/Instance/Authority.cs

```diff
@@ -117,7 +117,7 @@ private static void ValidateTypeMismatch(AuthorityInfo configAuthorityInfo, Auth
         }
 
 
-        public static Authority CreateAuthority(string authority, bool validateAuthority = false)
+        public static Authority CreateAuthority(string authority, bool validateAuthority = true)
```

Comment:
>validateAuthority = true [](start = 71, length = 24)

i think we want to leave the deafult value as false, otherwise it would be a breaking change. @jmprieur to confirm

Reply:
I can revert it back, changed it while testing and was not intending to commit it :)

Reply:
yes, it should stay as it was before. thanks. :)

Reply:
yes, needs to be reverted back :)

Reply:
reverted.